openforcefields/tests/test_water_models.py

```diff
@@ -90,6 +90,57 @@ def test_tip3p_fb(water_molecule):
         },
     )
 
+def test_tip4p_fb(water_molecule):
```

Comment:
@mattwthompson - Do you recall why we're skipping the tests for the other vsite water models (ex the [tip4p-ew test](https://github.com/openforcefield/openff-forcefields/pull/69/files#diff-b4f299b529f4606a8fbc265b06dc5d6c408f860781fece91fe0c6e68f4598e17R174)?). I feel like the skip there may be marking a gremlin in our testing machinery for multisite water models but I can't remember what it is.

Reply:
Scope creep in the original PR

Reply:
Thanks!